import-service/src/handlers/importFileParser.ts

```diff
@@ -0,0 +1,43 @@
+import { S3 } from 'aws-sdk';
+import { BUCKET_NAME } from '../constants';
+import { parse } from 'csv-parse';
+
+const importFileParser = (event: any) => {
+    const s3 = new S3({ region: 'eu-west-1' });
```

Comment:
it is better to pass s3 and other stuff as dependencies of lambda function.
for more details see here https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html#function-code